lighthouse-core/lib/cdt/generated/SourceMap.js

```diff
@@ -0,0 +1,509 @@
+// @ts-nocheck
```

Comment:
should we add this to some sort of assertion to make sure this always stays in sync with the script+devtools dep?

I could see someone bumping devtools for another reason and forgetting to update this.

Reply:
two options


1. run the build script in CI, fail if git is dirty.
1. print a comment with the devtools revision in this source file, and have a unit test that verifies the value in package.json is found in this file.

Reply:
I'd prefer 1 since it would catch changes to the script itself too, but not as big a deal.